docs/learn/learn-xcm.md

```diff
@@ -0,0 +1,299 @@
+designed to be a response message which can be sent from the recipient to the sender. Any kind of error
+handling should also be done on the recipient side.
+
+:::info
+
+XCM is not designed in a way where every system supporting the format is expected to be able to interpret
+any possible XCM message. Practically speaking, one can imagine that some messages will not have
+reasonable interpretations under some systems or will be intentionally unsupported.
+
+:::
+
+### Example Use-Cases
+
+- Request for specific operations to occur on the recipient system such as governance voting.
+- Enables single use-case chains e.g. Statemint/e as asset parachains
+- Optionally include payment of fees on a target network for requested operation.
+- Provide methods for various asset transfer models:
+  - **Remote Transfers**: control an account on a remote chain, allowing the local chain to have an
+    address on the remote chain for receiving funds and to eventually transfer those funds it
+    controls into other accounts on that remote chain.
+  - **Asset Teleportation**: movement of an asset happens by destroying it on one side and creating a clone
+    on the other side.
+  - **Reserve Asset Transfer**: there may be two chains that want to nominate a third chain, where
+    one includes a native asset that can be used as a reserve for that asset. Then, the derivative
+    form of the asset on each of those chains would be fully backed, allowing the derivative asset
+    to be exchanged for the underlying asset on the reserve chain backing it.
+
+Let's review two of these example asset transfer use cases: **Asset Teleportation** and **Reserve Asset Transfer**.
+
+### Asset Teleportation
+An asset teleport operation from a single source to a single destination. 
+
+![](../assets/cross-consensus/xcm-asset-teleportation.png)
+
+
+1. [InitiateTeleport](https://github.com/paritytech/xcm-format#initiateteleport)
+
+  The source gathers the assets to be teleported from the sending account and takes them out of the circulating supply, taking note of the total amount of assets that was taken out.
+
+2. [ReceiveTeleportedAsset](https://github.com/paritytech/xcm-format#receiveteleportedasset)
+
+  The source then creates an XCM instruction called `ReceiveTeleportedAssets` and puts the amount of assets taken out of circulation and the receiving account as parameters to this instruction. It then sends this instruction over to the destination, where it gets processed and new assets gets put back into circulating supply accordingly.
+
+3. [DepositAsset](https://github.com/paritytech/xcm-format#depositasset)
+
+  The destination then deposits the assets to the receiving account of the asset.
+
+### Reserve Asset Transfer
+When consensus systems do not have a established layer of trust over which they can transfer assets, they can opt for a trusted 3rd entity to store the assets. Eg. Statemine(Kusama) to Polkadot or Statemint(Polkadot) to Kusama. 
```

Comment:
The Polkadot-Kusama Bridge is not live yet. This example can be misleading. You could mention that if the remote chain is not on the trusted teleporter list, then Reserve Asset Transfer is the preferred method, where the asset stays on the original chain.

```suggestion
When consensus systems do not have a established layer of trust over which they can transfer assets, they can opt for a trusted 3rd entity to store the assets. 
```